src/plugins/graph/components/graph.tsx

```diff
@@ -1,6 +1,7 @@
 import {observer} from "mobx-react-lite";
 import {appConfig} from "../../../initialize-app";
-import React, {MutableRefObject, useEffect, useMemo, useRef} from "react";
+import React, {createContext, MutableRefObject, useContext, useEffect, useMemo, useRef} from "react";
+import { useId } from "@chakra-ui/react";
```

Comment:
Changing to `uniqueId` did not work.  `uniqueId` returns a new random ID on each render.  `useId` holds the generated id in state so that it is persistent across renders.

Reply:
Removed id generation from here